cloudflow-enterprise-components/values.yaml

```diff
@@ -19,7 +19,7 @@ enterprise-suite:
   busyboxImage: busybox
   esConsoleVersion: v1.4.10
   esMonitorVersion: v1.2.6
-  esGrafanaVersion: v0.5.0
+  esGrafanaVersion: v0.6.0
```

Comment:
This version fixes https://github.com/lightbend/console-home/issues/983 and bumps the Grafana version from 7.3.6 to 7.3.10.